concepts/observability-strategy/configurable-logging/01-groundwork.md

```diff
@@ -1,17 +1,16 @@
-# Current Situation and Motivation
+# Groundwork
 
-In the current shape of Kyma in 2021, the observability stack is focussing on providing opinionated solutions, working out of the box, to solve basic requirements for application operators. Consequently, it is not focussing on integration aspects in order to cover a broader and richer usage scenario.
+## Current Situation and Motivation
 
-![a](./assets/current_all.drawio.svg)
+![a](./assets/current.drawio.svg)
 
 In the diagram, you see that all three observability aspects (log, trace, metric) provide a preconfigured backend with visualisations. However, they don't provide a neutral and unified way to integrate backends outside of the cluster. The tracing stack provides no way to centrally push trace data to the outside. Logging can be configured much more flexibly and neutrally, however, the configuration must be done during installation so it isn't lost at the next Kyma upgrade process. Furthermore, it is not easy to use to mix and match different integrations, because you need to deal with one centralized configuration (the fluent-bit config).
 
-Integration (and with that, changing the focus away from in-cluster backends) is the key to open up the stack for a broad range of use cases. Users can simply bring their own backends if they already use a commercial offering or run their own infrastructure. The data can be stored outside the cluster in a managed offering, shared with the data of multiple clusters, away from any tampering or deletion attempt of a hacker, to name just a few.
+As outlined already in the [general strategy](../strategy.md), integration (and with that, changing the focus away from in-cluster backends) is the key to open up the stack for a broad range of use cases. Users can simply bring their own backends if they already use a commercial offering or run their own infrastructure. The data can be stored outside the cluster in a managed offering, shared with the data of multiple clusters, away from any tampering or deletion attempt of a hacker, to name just a few.
 
-This concept proposes how to open up to those new scenarios by making integration possible at runtime in a convenient way. For that, it focuses on the logging scenario. The other data types will be included at a later time.
-
-# Requirements
+This concept proposes how to open up to those new scenarios by making integration possible at runtime in a convenient way.
 
+## Requirements
 
 ### Basic backend configuration
 - Have a vendor-neutral agent layer that collects and ships the telemetry data, but does not permanently store it (as a backend)
```

Comment:
in line 26, "namespace" must be [capitalized](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#capitalized-terms).